internal/connect/command_custom_plugin_version.go

```diff
@@ -0,0 +1,58 @@
+package connect
+
+import (
+	"github.com/spf13/cobra"
+
+	connectcustompluginv1 "github.com/confluentinc/ccloud-sdk-go-v2/connect-custom-plugin/v1"
+
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type customPluginVersionOut struct {
+	Plugin              string   `human:"Plugin" serialized:"plugin"`
+	Version             string   `human:"Version" serialized:"version"`
+	Name                string   `human:"Name" serialized:"name"`
+	Description         string   `human:"Description" serialized:"description"`
+	Cloud               string   `human:"Cloud" serialized:"cloud"`
+	ConnectorClass      string   `human:"Connector Class" serialized:"connector_class"`
+	ConnectorType       string   `human:"Connector Type" serialized:"connector_type"`
+	SensitiveProperties []string `human:"Sensitive Properties" serialized:"sensitive_properties"`
+	VersionNumber       string   `human:"Version Number" serialized:"version_number"`
+	IsBeta              string   `human:"Beta" serialized:"is_beta"`
```

Comment:
Should this be a `bool`? The only options are "true" and "false", no? This matters since the YAML and JSON output will be different.